babel_runner.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/venv python3
+#!/usr/bin/env python3
```

Comment:
We're running this via `python babel_runner.py` in CI, shall we remove the shebang and executable bit?

```suggestion
```